csharp/src/Ice/ProxyFactoryMethods.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ZeroC, Inc.
+
+#nullable enable
+
+namespace Ice;
+
+public interface ProxyFactoryMethods<TProxy> : ObjectPrx where TProxy : class, ObjectPrx
+{
+    public new TProxy ice_oneway()
```

Comment:
"new" because there is no override / virtual with interface methods. And there is an existing abstract ice_oneway in ObjectPrx.